include/deal.II/fe/fe_system.h

```diff
@@ -1068,11 +1066,6 @@ class FESystem : public FiniteElement<dim,spacedim>
     ~InternalData();
 
     /**
-     * Flag indicating whether second derivatives shall be computed.
-     */
-    const bool compute_hessians;
```

Comment:
This is not the case yet; fe_poly_tensor still uses numerical differentiation to compute the hessian. I'm changing that class, and then all reference to numerical hessian computations can be removed.